src/typescript/frontend/src/components/pages/emojicoin/components/trade-history/index.tsx

```diff
@@ -15,12 +15,12 @@ import "./trade-history.css";
 const HARD_LIMIT = 500;
 
 const toTableItem = (value: Types.SwapEvent): TableRowDesktopProps["item"] => ({
-  ...getRankFromSwapEvent((value.balanceAsFractionOfCirculatingSupply ?? 0) * 100),
+  ...getRankFromEvent(value),
   apt: value.quoteVolume.toString(),
   emoji: value.baseVolume.toString(),
   date: new Date(Number(value.time / 1000n)),
   type: value.isSell ? "sell" : "buy",
-  price: value.avgExecutionPrice.toString(),
+  price: value.avgExecutionPriceQ64.toString(),
```

Comment:
Shouldn't this be converted to non-q64, or did I miss the conversion somewhere ?

Reply:
Nice catch, will convert it at this point (when calculating `price`)

Reply:
Fixed in 2d50728